tests/test_agents.py

```diff
@@ -242,7 +242,10 @@ async def test_get_directory_index_w_manifest(agent_test_settings: Settings) ->
 
         results = await index.query(query="who is Frederick Bates?")
         top_result = next(iter(results[0].docs.values()))
-        assert top_result.dockey == md5sum(abs_paper_dir / "bates.txt")
+
+        # note: we get every possible field from the manifest constructed in maybe_get_manifest,
```

Comment:
Let's get another test (or a `subtests.test` subtest) with a second manifest that leads to us not doing citation inference, something resembling the below

```csv
file_location,doi,title,citation
"bates.txt",,"Frederick Bates (Wikipedia article)",""
```